source/features/repo-age.tsx

```diff
@@ -0,0 +1,44 @@
+import React from 'dom-chef';
+import select from 'select-dom';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import * as icons from '../libs/icons';
+import {getRepoGQL, timeAgo} from '../libs/utils';
+
+async function getRepoCreationDate(): Promise<Date> {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			createdAt
+		}
+	`);
+
+	return new Date(repository.createdAt);
+}
+
+async function init(): Promise<void> {
+	const date = await getRepoCreationDate();
+	const {interval, unit} = timeAgo(date);
+	const element = (
+		<li title={`Repository created ${date.toDateString()}`}>
+			{icons.calendar()}
```

Comment:
I’m not a fan of this icon, it’s very generic about dates rather than the birth of something. Can we think of an alternative and more specific start/creation icon? 

For example would this make sense? https://octicons.github.com/icon/light-bulb/

Reply:
<img width="899" alt="Screen Shot 2019-11-21 at 11 50 24 AM" src="https://user-images.githubusercontent.com/3848317/69331476-232b1180-0c55-11ea-9a81-1cf53c677395.png">

I agree the calendar icon is very generic - I'm not sure which other octicon would be better though. The lightbulb here looks a little off to me.

Reply:
Would the regular repo icon be helpful?

```
[repo icon] 2 years old
```

Reply:
<img width="532" alt="Screen Shot 2019-11-23 at 2 56 31 AM" src="https://user-images.githubusercontent.com/3848317/69471179-e702c880-0d9c-11ea-96e8-6b2a098dede3.png">

I don't mind it myself and I don't see many other options unless Github adds a 🎂 octicon tomorrow.

Reply:
Works for me